cirq-web/requirements.txt

```diff
@@ -0,0 +1 @@
+IPython
```

Comment:
The only rationale I have for including it is that in a notebook users can use `display()` without actually needing to add IPython themselves. Looking back at it though I can see how it would be almost equally as easy for people to just import IPython and do it themselves. Changing now.

Reply:
Yeah, for notebook users, the notebook environment already typically includes IPython, as notebook is just a web version of IPython.